checker/tests/resourceleak/ACRegularExitPointTest.java

```diff
@@ -299,102 +299,4 @@ void testSubFoo2() {
     // :: error: required.method.not.called
     SubFoo f = new SubFoo();
   }
-
```

Comment:
I moved these test inputs to a new class `TernaryExpressions` to ease debugging test failures